changelog.md

```diff
@@ -1,20 +1,24 @@
 # Backpack changelog
 
 ## Unreleased
-_Nothing Yet_
+
+**Fixed:**
+- bpk-component-chip:
+- bpk-component-close-button:
+  - Fixed alignment issues in some cases (TODO: improve this)
```

Comment:
TODO?

Also should we mention other consumers of close-button (Banner alert etc)